webapp/src/ts/services/db-sync.service.ts

```diff
@@ -202,11 +202,15 @@ export class DBSyncService {
     return local
       .info()
       .then(info => currentSeq = info.update_seq)
-      .then(() => {
-        return local
-          .sync(remote)
-          .on('complete', (info) => telemetryEntry.recordSuccess(info))
-          .on('error', (err) => telemetryEntry.recordFailure(err, this.knownOnlineState));
+      .then(() => Promise.all([
+        local.replicate.to(remote),
+        local.replicate.from(remote),
+      ]))
```

Comment:
I'm in two minds about whether this should be in series or parallel. I think I prefer running in series to keep the background thread light. Why do you prefer running in parallel?

Reply:
I just kept it as `sync` does it, which starts both in parallel. 
I have no preference one way or the other.

Reply:
Ok. Consistency makes sense. Good to go!